readme.md

```diff
@@ -11,6 +11,12 @@ This is for advanced users. You probably want to use XO directly.
 $ npm install --save-dev eslint-config-xo
 ```
 
+For the `esnext` version you'll also need Babel's eslint parser and plugin:
```

Comment:
Can you linkify `parser` and `plugin` to the corresponding repos? In case the user wants to read more.